pkg/storer/internal/cache/cache.go

```diff
@@ -53,8 +61,7 @@ func (c *cacheEntry) Unmarshal(buf []byte) error {
 	}
 	newEntry := new(cacheEntry)
 	newEntry.Address = swarm.NewAddress(append(make([]byte, 0, swarm.HashSize), buf[:swarm.HashSize]...))
-	newEntry.Prev = internal.AddressOrZero(buf[swarm.HashSize : 2*swarm.HashSize])
-	newEntry.Next = internal.AddressOrZero(buf[2*swarm.HashSize:])
+	newEntry.AccessTimestamp = int64(binary.LittleEndian.Uint64(buf[swarm.HashSize:]))
```

Comment:
why is LittleEndian used here?
for the most part BigEndian is used (see internal/reserve) for having sorted.

Reply:
So the key creation is separate. The `ID()` function provides a string timestamp to enable the sorting. The way the value is stored doesnt really matter for the sorting.